packages/openapi3/test/works-for.ts

```diff
@@ -0,0 +1,64 @@
+import { describe } from "vitest";
+import { OpenAPIVersion } from "../src/lib.js";
+import {
+  checkFor,
+  diagnoseOpenApiFor,
+  emitOpenApiWithDiagnostics,
+  oapiForModel,
+  openApiFor,
+  openapiWithOptions,
+} from "./test-host.js";
+
+export const OpenAPISpecHelpers: Record<OpenAPIVersion, SpecHelper> = {
+  "3.0.0": {
```

Comment:
How about making a function to reuse like the follwing, we might need to add more versions in the future
```ts
createHelper(version: OpenAPIVersion) => {
    return {... { ...options, "openapi-versions": [version.toString()] } ...}
}
```